modules/apps/foundation/file-uploads/file-uploads-configuration-test/build.gradle

```diff
@@ -0,0 +1,14 @@
+dependencies {
+	provided project(":apps:foundation:file-uploads:file-uploads-configuration")
```

Comment:
change `provided` to `testIntegrationCompile`. see https://github.com/liferay/liferay-portal/blob/master/modules/apps/wsrp/wsrp-test/build.gradle